applications/GeoMechanicsApplication/tests/test_transient_thermal.py

```diff
@@ -0,0 +1,157 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+class KratosGeoMechanicsTransientThermalTests(KratosUnittest.TestCase):
+    """
+    This class contains benchmark tests which are checked with the analytical solution
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+    
+    def test_thermal_heat_flux_2D3N(self):
+        test_name = 'test_thermal_heat_flux_2D3N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name + '.gid'))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
+        temp = temperature[37]
+        self.assertAlmostEqual(28.04411163544510063559, temp)
+
+    def test_thermal_heat_flux_2D6N(self):
+        test_name = 'test_thermal_heat_flux_2D6N'
+        file_path = test_helper.get_file_path(os.path.join('test_thermal_element', test_name + '.gid'))
+        simulation = test_helper.run_kratos(file_path)
+        temperature = test_helper.get_temperature(simulation)
+        temp = temperature[57]
+        self.assertAlmostEqual(28.04411163544510063559, temp)
+
+    def test_thermal_heat_flux_2D10N(self):
```

Comment:
To me it is very hard to see if these tests are correct. Is there an analytical solution here, or is it a regression tests. Do you know @mnabideltares?